geosolutions-it/MapStore2
# Identify Mode not persisted when map saved (#5566)

## Description
This PR ensures that when a user selects identify mode from BurgerMenu > Settings as either Click or Hover, that mode is persisted with the data resource data.

**Please check if the PR fulfills these requirements**


**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [x] Other... Enhancement to existing feature



# Issue

**What is the current behavior?**

#5566 

**What is the new behavior?**
Identify tool mode is persisted. For example if a user selects Hover, it's persisted with that particular map resource

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)



## Other useful information
How to test thing feature
- Import a map with layers in MapStore
- Click Burger menu > Settings
- Click on `Trigger event for the display of the information sheet` and select either Hover or Click and close that modal
- Click on BurgerMenu > Save
- If you navigate away, the hover mode (if was selected) should still be activated
